### ISSUE TITLE
ANSI codes not processed by Python 3.8 REPL on Windows 10

### ISSUE THREAD

**Issue by MinchinWeb** [tartley/colorama#233]
(I don't know whether this is a Python issues or a colorama issue, so I cross-posted it to the Python [bug tracker](https://bugs.python.org/issue38552) as well.)

I have a Python startup file that colorizes my prompt. This worked on Python 3.7, but breaks on Python 3.8. I'm on Windows using Powershell 6.2.3, but `cmd` does the same.

Maybe related, but the colorization also failed on Python 3.7 if a virtual environment was active.

I am using `colorama` to provide the ANSI color codes.

The startup files:

```
import sys
import colorama

colorama.init()

_GREEN = colorama.Fore.GREEN
_YELLOW = colorama.Fore.YELLOW
_RESET = colorama.Style.RESET_ALL

print("setting prompt, {}, {}, {}.".format(_GREEN, _YELLOW, _RESET))
sys.ps1 = "{}>>> {}".format(_GREEN, _RESET)
sys.ps2 = "{}... {}".format(_YELLOW, _RESET)
print()
```
![python_colored_prompt](https://user-images.githubusercontent.com/1548809/67232471-a9f29180-f3fe-11e9-88e4-96ad8cf7505e.png)

**Comment by jsbueno-geru** [comment]
I am having the issue here - I've just made a fresh virtualbox Windows 10 start + python3.8 - to start windows support to my lib, just to findout that colorama behaves as a no-op on python3.8  (it strangely works if I send a manual print from the repl, but not if the print code is in an imported Python file).

I will try later with an older Python

**Comment by jsbueno-geru** [comment]
FOund out the problem - my library was actually printing the ANSI sequences in separated chunks - 
like in `print("\x1b[", "2J", sep="", end="")` - The current colorama code does not supporting splitting an ANSI sequence in different write chunks.  

This might be the cause for other issues - but has nothing to do with Python3.8 or windows10

**Comment by MinchinWeb** [comment]
I can't reproduce this today: Python 3.8.6 (or 3.9.5) with PowerShell 7.1.3 on Windows 10 with Windows Terminal. Maybe it got fixed by a bugfix release of Python 3.8?
